src/ciceroscm/two_layer_ocean.py

```diff
@@ -0,0 +1,85 @@
+"""
+2 layer model
+"""
+
+import numpy as np
+
+
+class TwoLayerOceanModel:
+    """
+    Two layer Model with 2 thermal timescales.
+    """
+
+    def __init__(self, params):
+        """
+        Initialize with parameters for multiple thermal timescales.
+
+        Parameters
+        ----------
+        params : dict
+            Dictionary containing model parameters:
+            - lambda: Climate feedback parameter (W/m^2/K)
+            - c_fast: Heat capacity of the fast-response layer (J/m^2/K)
+            - c_slow: Heat capacity of the slow-response layer (J/m^2/K)
+            - k: Coupling coefficient between fast and slow layers (W/m^2/K)
+        """
+        self.lambda_ = params.get("lambda", 3.74 / 3)
```

Comment:
Some common definitions of constants might make sense here. The old udm has SEC_Day and DAY_YEAR, maybe both should go to some common constants definition in _utils of pub_utils, anyways this is a bit of a mess...